CONTRIBUTING.md

```diff
@@ -96,19 +96,20 @@ The [Neuron Development Topics](https://neuronsimulator.github.io/nrn/dev/index.
 
 ### Code Formatting
 
-Currently we have enabled CMake code formatting using [cmake-format](https://github.com/cheshirekow/cmake_format). Before submitting PR, if you have changed any CMake build related code, make sure to run cmake-format as below:
+Currently we have enabled CMake and Clang code formatting using [cmake-format](https://github.com/cheshirekow/cmake_format) and [clang-format](https://clang.llvm.org/docs/ClangFormat.html). Before submitting a PR, make sure to run cmake-format as below:
```

Comment:
cmake-format => cmake-format and clang-format

Reply:
fixed